File: Taejin/DS_Module/QueueExample.py
# Queue_Example.py
import collections

# Declaration
q = collections.deque()

# Enqueue values
q.append( 1 )
q.append( 2 )
q.append( 3 )

# Dequeue values
print( q.popleft( ) )

# Front of queue
print( q[0] );

File: Taejin/DS_Module/QueueExample.cpp
// Queue_Example.cpp
#include <iostream>
#include <queue>

using namespace std;

int main(void) {
	// Declaration
	queue<int> q;

	// Enqueue
	q.push( 1 );
	q.push( 2 );
	q.push( 3 );

	// Return front of queue
	cout << q.front() << endl;

	// Dequeue
	q.pop();

	return 0;
}